### ISSUE TITLE
Does the simulator have an in built DBW control system ?

### ISSUE THREAD

**Issue by Mojojojooo** [lgsvl/simulator#1019]
1.Does the car have an inbuilt controller ? if yes can make any changes to it.
2. Even if has a built in controller can it be used with a personal external controller ?

**Comment by EricBoiseLGSVL** [comment]
@Mojojojooo Yes, we have FMU support and we also have a VehicleSMI dynamics class that you can edit.  You can also write your own dynamics class as long as it uses the interface for dynamics.  Not quite sure what you mean on 2. but if you are talking about external dyamics applications, then yes.  We are currently working on supporting this but this won't be released for a while.  Hope this helps
